.editorconfig

```diff
@@ -0,0 +1,23 @@
+# EditorConfig is awesome:http://EditorConfig.org
```

Comment:
The Razor editorconfig has a bunch of settings that are already the VS default. IIUC the only non-default we use is sorting System.* usings first.